src/utils/generateFlowDiagramData.js

```diff
@@ -97,14 +98,25 @@ const generateFlowDiagramData = (data) => {
     { flowData: {}, mostRecentYear: -Infinity, mostRecentMonth: -Infinity }
   );
 
-  Object.values(flowData).forEach((item) => {
-    const hint = item.isNotAvailable
+  Object.values(flowData).forEach((metric) => {
+    const hint = metric.isNotAvailable
       ? null
-      : generateHint(mostRecentYear, mostRecentMonth, item.item);
+      : generateHint(mostRecentYear, mostRecentMonth, metric.item);
 
     if (hint) {
       // eslint-disable-next-line no-param-reassign
-      item.hint = hint;
+      metric.hint = hint;
+    }
+
+    const { item } = metric;
+
+    const isTooStale = metric.isNotAvailable
```

Comment:
Let's add a brief comment right above this saying, "A metric is too stale if its month and year are earlier than the month and year we are comparing metrics against"

Reply:
Done